rsts/deployment/aws/manual.rst

```diff
@@ -407,11 +407,33 @@ Time for Helm
 
 Installing Flyte
 -----------------
-#. Clone the Flyte repo
 
-.. code-block:: bash
+#. Add the flyte chart repo to helm
+
+  .. code-block::
```

Comment:
In the rendered view this like like a quoted string... I think it has an extra space in the beginning of the line... Maybe this line or maybe the next line is the problem